BreadCrumb Control/BreadCrumb Control/BreadCrumb.swift

```diff
@@ -304,10 +315,9 @@ public class CBreadcrumbControl: UIScrollView {
         super.layoutSubviews()
         
         var cx: CGFloat = 0  //kStartButtonWidth
-        for view: UIView in _itemViews
-        {
-            let s: CGSize = view.bounds.size
-            view.frame = CGRect(origin:CGPoint(x:cx, y:0), size:CGSize(width:s.width, height:s.height))
+        for view: UIView in _itemViews {
+            let s = view.bounds.size
+            view.frame = CGRect(origin: CGPoint(x: cx, y: 0), size: CGSize(width: s.width, height: s.height))
             cx += s.width
         }
         initialSetup( refresh: true)
```

Comment:
`initialSetup( refresh: true)` to `initialSetup(refresh: true)`. There's 3 other instances in `BreadCrumb.swift`